src/inject/generic/core/lyrics.js

```diff
@@ -52,9 +54,28 @@ window.addEventListener('load', () => {
     animate = false;
   };
 
+  const scrollHandler = (state) => {
+    const lyricsP = $('#lyrics p');
+    if (state) { // enable auto scroll
+      lyricsP.attr('scroll', true);
+      animate = true;
+      timeHandler(remote.getGlobal('PlaybackAPI').data.time);
```

Comment:
We should call `clearTimeout(animationTimer)` before this line

IPC events are asynchronous, we don't want to run into any problems 😆